openpgp.d.ts

```diff
@@ -41,14 +41,14 @@ export class Key {
   public isRevoked(): Promise<boolean>;
   public revoke(reason: { flag?: enums.reasonForRevocation; string?: string; }, date?: Date): Promise<Key>;
   public getRevocationCertificate(): Promise<Stream<string> | string | undefined>;
-  public getEncryptionKey(keyid?: Keyid | null, date?: Date, userId?: UserId | null): Promise<Key | SubKey | null>;
-  public getSigningKey(keyid?: Keyid | null, date?: Date, userId?: UserId | null): Promise<Key | SubKey | null>;
-  public getKeys(keyId?: Keyid): (Key | SubKey)[];
+  public getEncryptionKey(keyid?: KeyId, date?: Date | null, userId?: UserId): Promise<Key | SubKey | null>;
+  public getSigningKey(keyid?: KeyId, date?: Date | null, userId?: UserId): Promise<Key | SubKey | null>;
```

Comment:
FWIW, these can't return `null` anymore, they throw if there is no encryption/signing key.